.github/workflows/run-preview.yml

```diff
@@ -9,10 +9,11 @@ jobs:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
-      - run: yarn
-      - run: yarn build
+      - uses: volta-cli/action@v1
+      - run: npm ci --ignore-scripts
+      - run: npm run build
```

Comment:
I don't think this will build the action itself. If we are using the "local" version of the action, we probably want to build that.

Reply:
oh it's not to build the action but the rest of covector packages. i left it like this so that we don't forget to build and publish the updated action.

Reply:
> so that we don't forget to build and publish the updated action

I feel like I might be misunderstanding something here. I don't understand the use case. Could you elaborate?

Reply:
oh i guess if we're not building the action `pkg -w action`, we don't need to run `ci` or `build`.

Reply:
> > so that we don't forget to build and publish the updated action
> 
> I feel like I might be misunderstanding something here. I don't understand the use case. Could you elaborate?

Any changes I make to the action, if I don't build locally and push the updated `dist/` to github, the next release will still have the old action. So building the action inside the workflows for the covector repository seems like it would lead to an oversight.

Reply:
The `.changes/config.json` has a `preversion` script for this exact case, haha. It will build the action and any changes will show up in the Version Packages PR.

https://github.com/jbolda/covector/blob/main/.changes/config.json#L105

This is also partly because we are using `@effection/node` which uses `ctrlc-windows` which is built per platform. So the `dist` that I create on my windows machine will fail as the actions are (generally) run on Linux. If I get a free minute, it might be nice to try to address that (and switch to the N-API).

Reply:
ohh i see 👌

Reply:
Yeeep, I definitely forgot multiple times and just decided that was the most failsafe :)